data/json/items/ammo.json

```diff
@@ -1327,5 +1327,49 @@
     "count": 50,
     "ammo_type": "tinder",
     "effects": [ "NEVER_MISFIRES" ]
+  },
+  {
+    "type": "AMMO",
+    "id": "RPG_die",
+    "price": 10,
+    "name": "RPG die",
+    "name_plural": "RPG dice",
+    "symbol": "=",
+    "color": "light_gray",
+    "looks_like": "marble",
+    "description": "A die used to play various role-playing games.",
+    "material": "plastic",
+    "volume": 1,
+    "weight": 1,
+    "ammo_type": "pebble",
+    "damage": 1,
+    "range": 20,
+    "dispersion": 15,
+    "loudness": 0,
+    "count": 7,
+    "effects": [ "NEVER_MISFIRES" ],
+    "use_action": "RPGDIE"
+  },
+  {
+    "type": "AMMO",
+    "id": "metal_RPG_die",
+    "price": 50,
+    "name": "metal RPG die",
+    "name_plural": "metal RPG dice",
+    "symbol": "=",
+    "color": "light_gray",
+    "looks_like": "marble",
+    "description": "A metal die used to play various role-playing games",
+    "material": "zinc",
```

Comment:
This material does not exist

Reply:
Right, I made another PR adding it, and removed it from this one. I'll put it back in this one and just mention that in both PRs.